Devscord.DiscordFramework/Services/ReadMessagesHistoryService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+
+namespace Devscord.DiscordFramework.Services
+{
+    public class ReadMessagesHistoryService
+    {
+        public async Task<IEnumerable<(Contexts contexts, DiscordRequest request, ulong messageId)>> ReadMessagesAsync(DiscordServerContext server, ChannelContext channelContext, int limit)
```

Comment:
może to powinno być osobnym obiektem 
taki potężny tuple nie wygląda przejrzyście 
(Contexts contexts, DiscordRequest request, ulong messageId)